components/Card.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```

Comment:
Please use the Card component available in the branch and modify it if necessary.